fighthealthinsurance/utils.py

```
@@ -477,14 +504,24 @@ async def execute_critical_optional_fireandforget(
-            yield result
+            try:
+                result: T = await task
+                # Yield each result immediately for streaming
+                yield result
+            except asyncio.CancelledError:
+                logger.debug("Task was cancelled, skipping result")
+                continue
             if required_tasks_finished >= len(required):
                 logger.debug("All done with required tasks")
                 break
     except asyncio.TimeoutError as e:
```

Ruff (0.11.9)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)